src/base/kaldi-error.cc

```diff
@@ -18,156 +19,122 @@
 // limitations under the License.
 
 #ifdef HAVE_EXECINFO_H
-#include <execinfo.h>  // To get stack trace in error messages.
-// If this #include fails there is an error in the Makefile, it does not
-// support your platform well. Make sure HAVE_EXECINFO_H is undefined, and the
-// code will compile.
-#ifdef HAVE_CXXABI_H
-#include <cxxabi.h>  // For name demangling.
-// Useful to decode the stack trace, but only used if we have execinfo.h
-#endif  // HAVE_CXXABI_H
+  #include <execinfo.h>  // To get stack trace in error messages.
+  // If this #include fails there is an error in the Makefile, it does not
+  // support your platform well. Make sure HAVE_EXECINFO_H is undefined, 
+  // and the code will compile.
+  #ifdef HAVE_CXXABI_H
+    #include <cxxabi.h>  // For name demangling.
```

Comment:
please don't indent the #ifdefs, I couldn't find any example in the Google C++ style guide of doing this.